admin/app/views/commercial/commercialMenu.scala.html

```diff
@@ -112,6 +112,9 @@
                     <ul>
                         <li><a href="https://github.com/guardian/commercial-tools">Commercial tools repo</a></li>
                         <li><a href="https://dfp-playground.appspot.com/59739193">DFP API playground</a></li>
+                        <li><a href="http://kibana.gu-web.net/app/kibana#/dashboard/Commercial-Feeds?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-4h,mode:quick,to:now))&_a=(filters:!(),options:(darkTheme:!t),panels:!((col:1,id:'Failing-Commercial-Feeds-(Fetch)',panelIndex:2,row:1,size_x:6,size_y:4,type:visualization),(col:7,id:'Failing-Commercial-Feeds-(Parse)',panelIndex:3,row:1,size_x:6,size_y:4,type:visualization),(col:1,columns:!(commercial-feed,commercial-feed-action,message),id:'Commercial-Feeds-(Failing)',panelIndex:4,row:5,size_x:12,size_y:3,sort:!('@timestamp',desc),type:search)),query:(query_string:(analyze_wildcard:!t,query:'*')),title:'Commercial%20Feeds',uiState:(P-2:(spy:(mode:(fill:!f,name:!n)))))">
```

Comment:
You can have kibana generate a short link :smile:
![Uploading Screen Shot 2016-07-05 at 10.33.04.png…]()